### PR TITLE
[4.0] Add modules to location in new menu

### PR DESCRIPTION
**Pull request by wilsonge**
Adds the menu items for the modules in the menu per the invision specification

### CODE REVIEW

File: administrator/language/en-GB/en-GB.mod_menu.ini [hunk 1]
```diff
@@ -35,7 +35,8 @@ MOD_MENU_CONTROL_PANEL="Control Panel"
 MOD_MENU_EXTENSIONS_EXTENSIONS="Extensions"
 MOD_MENU_EXTENSIONS_EXTENSION_MANAGER="Manage"
 MOD_MENU_EXTENSIONS_LANGUAGE_MANAGER="Language(s)"
-MOD_MENU_EXTENSIONS_MODULE_MANAGER="Modules"
+MOD_MENU_EXTENSIONS_MODULE_MANAGER_SITE="Frontend Modules"
+MOD_MENU_EXTENSIONS_MODULE_MANAGER_ADMINISTRATOR="Backend Modules"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.mod_menu.ini]: we have always use site and administrator in the past

File: administrator/components/com_menus/presets/joomla.xml [hunk 2]
```diff
@@ -60,6 +60,18 @@
...
+				type="component"
+				element="com_modules"
+				link="index.php?option=com_modules&amp;client_id=0"
+		/>
+		<menuitem
+				title="MOD_MENU_EXTENSIONS_MODULE_MANAGER_ADMINISTRATOR"
+				type="component"
+				element="com_modules"
+				link="index.php?option=com_modules&amp;client_id=1"
+		/>
```
Comment (brianteeman) [inline administrator/components/com_menus/presets/joomla.xml:74]: the tabs on this file are completely off - not just this edit

**Review by brianteeman — CHANGES_REQUESTED** [review]

File: administrator/components/com_menus/presets/joomla.xml (hunk 2 above)
Comment (wilsonge) [reply administrator/components/com_menus/presets/joomla.xml:74]: Agreed. Let’s fix in a future PR in one go? This is consistent with test of the file

File: administrator/language/en-GB/en-GB.mod_menu.ini (hunk 1 above)
Comment (wilsonge) [reply administrator/language/en-GB/en-GB.mod_menu.ini]: i was staying to the designs - but you're right should be done all at once if we're going to

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.mod_menu.ini]: I am always right - when will you learn that - it will save you so much time

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 43077dabb487bcd297c8b5733056ba2a26aba03a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21032">issues.joomla.org/tracker/joomla-cms/21032</a>.</sub>

**Comment by ghost** [comment]
Changed Title so its clear its about 4.0.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21032">issues.joomla.org/tracker/joomla-cms/21032</a>.</sub>

**Review by brianteeman — APPROVED** [review]

**Pull request merged**